jobbergate-agent/tests/jobbergate/test_submit.py

```diff
@@ -714,6 +725,13 @@ def _mocked_mark_as_submitted(job_submission_id: int, slurm_job_id: int):
         [
             mocker.call(1, 11, SlurmJobData(job_state="RUNNING", job_info="{}")),
             mocker.call(2, 22, SlurmJobData(job_state="RUNNING", job_info="{}")),
+            mocker.call(4, 44, SlurmJobData(job_state="RUNNING", job_info="{}")),
         ]
     )
-    assert mock_mark.call_count == 2
+    assert mock_mark.call_count == 3
+
+    assert cached_submissions[1].exists() is False
+    assert cached_submissions[2].exists() is True
+    assert cached_submissions[2].read_text() == "22"
+    assert cached_submissions[3].exists() is False
+    assert cached_submissions[4].exists() is False
```

Comment:
You might consider writing these assertions as:
```suggestion
    assert not cached_submissions[1].exists()
    assert cached_submissions[2].exists()
    assert cached_submissions[2].read_text() == "22"
    assert not cached_submissions[3].exists()
    assert not cached_submissions[4].exists()
```
I think it reads a bit nicer.